docs-developer/docker_setup.md

```diff
@@ -0,0 +1,9 @@
+# Using docker-compose to set up your environment
```

Comment:
This seems like a really small doc to separate out. Any reason not to just move it into local_preview.md?

Reply:
Since it's used for development, I'm not sure if calling it a "preview" might make it seem like it's not for active updating?